pkg/controller/deployment/related_images.go

```diff
@@ -0,0 +1,20 @@
+package deployment
+
+import "os"
+
+const (
+	environmentalVariableCertManagerRelatedImage = "RELATED_IMAGE_CERT_MANAGER"
```

Comment:
this constant is not used elsewhere, it's fine to inline.

Reply:
This variable is quite significant from the prod perspective. I'd like it to stay as `const` so that it screams as you whenever you open this file.